resources/i18n/fr.json5

```diff
@@ -1,3220 +1,1596 @@
 {
   
   // "404.help": "We can't find the page you're looking for. The page may have been moved or deleted. You can use the button below to get back to the home page. ",
-  // TODO New key - Add a translation
-  "404.help": "We can't find the page you're looking for. The page may have been moved or deleted. You can use the button below to get back to the home page. ",
-  
+  "404.help": "La page que vous recherchez n'a pas pu être localisée. Cette page pourrait avoir été déplacée ou supprimée. Vous pouvez utiliser le bouton ci-dessous pour revenir sur la page d'accueil.",
   // "404.link.home-page": "Take me to the home page",
-  // TODO New key - Add a translation
-  "404.link.home-page": "Take me to the home page",
-  
+  "404.link.home-page": "Retour sur la page d'accueil",
   // "404.page-not-found": "page not found",
-  // TODO New key - Add a translation
-  "404.page-not-found": "page not found",
-  
-  
+  "404.page-not-found": "Page introuvable",
   
   // "admin.registries.bitstream-formats.create.failure.content": "An error occurred while creating the new bitstream format.",
-  // TODO New key - Add a translation
-  "admin.registries.bitstream-formats.create.failure.content": "An error occurred while creating the new bitstream format.",
-  
+  "admin.registries.bitstream-formats.create.failure.content": "Une erreur s'est produite lors de l'ajout du nouveau format Bitstream.",
```

Comment:
I would suggest using "fichier" for the translation of "bitstream"

Reply:
As discussed separately in Slack, we would prefer to stick to the English wordings for the generic DSpace terms as we can more easily refer to the DSpace documentation.